app/modules/auth.py

```
@@ -0,0 +1,17 @@
+from config import settings
+import hashlib
+
+def authenticate(username, password):
+    hashed_password = hashlib.md5(password.encode()).hexdigest()
```

Ruff (0.8.2)

Local variable `hashed_password` is assigned to but never used

Remove assignment to unused variable `hashed_password`

(F841)